src/app/Redemptions/components/RedeemPointsDialogComponent.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from '@material-ui/core/Select';
+import Dialog from '@material-ui/core/Dialog';
+import CloseIcon from '@material-ui/icons/Close';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+
+import { ButtonComponent } from '../../common/components';
+
+import centers from '../contants';
+
+const RedeemPointsDialogComponent = ({
+  open,
+  date,
+  errors,
+  reason,
+  center,
+  points,
+  usdValue,
+  onClose,
+  onChange,
+  handleRedemptionSubmit,
+}) => (
+  <Dialog
+    disableEscapeKeyDown
+    onClose={onClose}
+    open={open}
+    scroll='body'
+    PaperProps={{
```

Comment:
@Chris-Maina What does this `PaperProps` do?

Reply:
It helps you style the `Paper` element that comes with the `Dialog` component.